src/_data/audioworklet_data.yaml

```diff
@@ -42,6 +42,9 @@
     - title: Audio Worklet, SharedArrayBuffer, and Worker
       description: For high performance large-scale audio applications
       href: design-pattern/shared-buffer/
+    - title: Rainfly 
```

Comment:
I think `landing_data.yml` is the right place for this entry

Reply:
sounds good! I wasn't sure if this should be AudioWorklet specific but can move it over!